src/mrinufft/trajectories/display3D.py

```diff
@@ -0,0 +1,144 @@
+    allowing for easier viewing of the trajectory.
+    The gridding process can be carried out to reflect the sampling density,
+    sampling time, inversion time, k-space holes, gradient strengths, or slew rates.
+    Please check `grid_type` parameter to know the benefits of each type of gridding.
+    During the gridding process, the values corresponding to various samples within the
+    same voxel get averaged.
+
+    Parameters
+    ----------
+    trajectory : ndarray
+        The input array of shape (N, M, D), where N is the number of shots and M is the
+        number of samples per shot and D is the dimension of the trajectory (usually 3)
+    shape : tuple
+        The desired shape of the gridded trajectory.
+    grid_type : str, optional
+        The type of gridded trajectory to compute. Default is "density".
+        It can be one of the following:
+        "density" : Get the sampling density in closest number of samples per voxel.
+            Helps understand suboptimal sampling, by showcasing regions with strong
+            oversampling.
+        "time" : Showcases when the k-space data is acquired in time.
+            This is helpful to view and understand off-resonance effects.
+            Generally, lower off-resonance effects occur when the sampling trajectory
+            has smoother k-space sampling time over the k-space.
+        "inversion" : Relative inversion time at the sampling location. Needs
+            turbo_factor to be set. This is useful for analyzing the exact inversion
+            time when the k-space is acquired, for sequences like MP(2)RAGE.
+        "holes": Show the k-space holes within a ellipsoid of the k-space.
+        "gradients": Show the gradient strengths of the k-space trajectory.
+        "slew": Show the slew rate of the k-space trajectory.
+    osf : int, optional
+        The oversampling factor for the gridded trajectory. Default is 1.
+    backend : str, optional
+        The backend to use for gridding. Default is "gpunufft".
+        Note that "gpunufft" is anyway used to get the `pipe` density internally.
+    traj_params : dict, optional
+        The trajectory parameters. Default is None.
+        This is only needed when `grid_type` is "gradients" or "slew".
+        The parameters needed include `img_size` (tuple), `FOV` (tuple in `m`),
+        and `gamma` (float in kHz/T) of the sequence.
+        Generally these values are stored in the header of the trajectory file.
+    turbo_factor : int, optional
+        The turbo factor when sampling is with inversion. Default is 176, which is
+        the default turbo factor for MPRAGE acquisitions at 1mm whole
+        brain acquisitions.
+    elliptical_samp : bool, optional
+        Whether the trajectory uses elliptical sampling. Default is True.
+        This is useful while analyzing the k-space holes, especially if the k-space
+        trajectory is expected to be elliptical sampling of k-space
```

Comment:
Why is this information used in this function ?

Reply:
Cause most of sampling patterns in Non-Cartesian is elliptical and not on a cube. If you search for k-space holes without it set, you will just see large holes between cube and ellipse :P

Reply:
I know, but please state it in the documentation. Users have no idea what they would be doing by toggling this on or off

Reply:
@chaithyagr

Reply:
```suggestion
        Whether the k-space corners should be expected to be covered
        or ignored when `grid_type` is "holes". Ignoring them with `True`
        corresponds to trajectories with spherical/elliptical sampling.
        Default is `True`.
```
Here is a proposition to clarify my point